src/pages/index.mdx

```diff
@@ -105,3 +105,9 @@ When you commit to leveling up with a year of egghead, you will get instant acce
 You can use your new skills to ship a new project, negotiate a raise, or just for fun. But whatever you do, don't wait until this deal's over.
 
 [Click here to become an egghead member!](https://egghead.io/pricing)
+
+import DefaultLayout from '../layouts/index'
```

Comment:
my feeling generally is that this is a fine way to handle templates

struggled to avoid needing an `import` for the default template. That'd make it a lot nicer.

Reply:
I took a stab at it, but failed. I think we'd have to customize babel in next.config.js `withMDX` to accept `https://babeljs.io/docs/en/babel-plugin-proposal-export-default-from`, but my effort was in vain, so 🤷‍♂️

Reply:
ya that was my observation too and concluded that staying "stock" is solid goal where possible